crates/tests/properties/tests/test.rs

```diff
@@ -45,7 +45,7 @@ fn test() -> Result<()> {
         assert!(b.GetCount()? == 123);
 
         let c: IPropertyStoreCapabilities = b.cast()?;
-        c.IsPropertyWritable(std::ptr::null())?;
+        c.IsPropertyWritable(&PROPERTYKEY::default())?;
```

Comment:
This seems different than it was before. Why isn't `None` being passed?

Reply:
This parameter is not optional.